dev/remsh

```diff
@@ -26,4 +25,55 @@ fi
 NAME="remsh$$@$HOST"
 NODE="node$NODE@$HOST"
 COOKIE=monster
-erl -name $NAME -remsh $NODE -setcookie $COOKIE -hidden
+
+display_help () {
+    cat << EOF
+Usage: $basename [OPTION]
+
+The $basename command is responsible for generating the build
+system for Apache CouchDB.
+
+Options:
+
+  -h | --help                 display a short help message and exit
+  --tcp | --TCP               enable TCP distribution
+  --tls | --TLS               enable TLS distribution
+EOF
+}
+
+parse_opts() {
+    while :; do
+        case $1 in
+            -h|--help)
+                display_help
+                exit
+                ;;
+
+            --tcp|--TCP)
+                erl -name $NAME -remsh $NODE -setcookie $COOKIE -hidden
+                shift
+                continue
+                ;;
+
+            --tls|--TLS)
+                erl -name $NAME -remsh $NODE -setcookie $COOKIE -hidden -proto_dist inet_tls -ssl_dist_optfile "${rootdir}/ssl_dist.conf"
```

Comment:
Note that remsh ends up in `$release_top/bin/remsh` in the release 

Run `make release` then check in `./rel/couchdb/`.  Perhaps we look for in `$release_top/etc/couchdb_ssl_dist.conf` or, we just say that `remsh.tls` always requires an `--tls-dist-conf` with a file path. With a nice option to pick it up from the environment var `COUCHDB_TLS_DIST_CONF`